src/main/java/main/movie/Main.java

```diff
@@ -58,5 +62,28 @@ public static void main(String[] args) {
         System.out.println(movieSessionService
                 .findAvailableSessions(2L, LocalDate.of(2020,
                         10, 22)));
+
+        UserService userService
+                = (UserService) injector.getInstance(UserService.class);
+
+        User user = new User();
+        user.setPassword("1234");
+        user.setEmail("myemdfail");
+        userService.add(user);
```

Comment:
let's register a new user via AuthenticationService
(and add to `register()` method registering the new shopping cart as well)